projects/collection-editor-library/src/lib/components/fancy-tree/fancy-tree.component.ts

```diff
@@ -88,7 +88,7 @@ export class FancyTreeComponent implements OnInit, AfterViewInit, OnDestroy {
     const data = this.nodes.data;
     this.nodeParentDependentMap = this.editorService.getParentDependentMap(this.nodes.data);
     let treeData;
-    if (_.get(this.editorService, 'editorConfig.config.renderTaxonomy') === true) {
+    if (_.get(this.editorService, 'editorConfig.config.renderTaxonomy') === true && _.isEmpty(_.get(this.editorService.collectionTreeNodes, 'data.children'))) {
```

Comment:
we do not need `this.editorService.collectionTreeNodes` in  `&& _.isEmpty(_.get(this.editorService.collectionTreeNodes, 'data.children'))`. we can use `this.nodes.data` right?